.all-contributorsrc

```diff
@@ -284,6 +284,16 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "dsingleton",
+      "name": "David Singleton",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/63201?v=4",
+      "profile": "http://dsingleton.co.uk",
+      "contributions": [
+        "doc"
+      ]
     }
-  ]
+  ],
+  "repoType": "github"
```

Comment:
I was confused by this bit of the diff, but I think it could be because [`all-contributors-cli` was upgraded](https://github.com/prettier/prettier-atom/commit/0d62f94d17ea8ae8d5329e4a8f57aca01b40ec92) since the last contributor was added.

Reply:
Ah yes that's probably what it is, no worries